Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -132,15 +132,21 @@
   components:
   - type: RampingStationEventScheduler
 
+- type: entity
+  id: LongSurvivalStationEventScheduler
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: RampingStationEventScheduler
+    shiftLengthOffset: -120
+
 - type: entity
   id: HellshiftStationEventScheduler
   parent: BaseGameRule
   noSpawn: true
   components:
   - type: RampingStationEventScheduler
-    chaosModifier: 4 # By default, one event each 30-10 seconds after two hours. Changing CVars will cause this to deviate.
-    startingChaosRatio: 0.025 # Starts as slow as survival, but quickly ramps up
-    shiftLengthModifier: 2.5
+    shiftChaosModifier: 4 #30 minute HELL SHIFT
```

Comment:
do not

Reply:
Do me a favor, look over the available settings(and funny graph) and tell me what you'd like Hellshift's new variables to be. :)

Reply:
it's meant to start slow to give the crew time to prepare for what's coming